app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,25 @@
+class WebhooksController < ApplicationController
+  include ShopifyApp::WebhookVerification
+
+  class ShopifyApp::MissingWebhookJobError < StandardError; end
+
+  def receive
+    job_args = {shop_domain: shop_domain, webhook: webhook_params}
+    webhook_job_klass.perform_later(job_args)
```

Comment:
I thought I said this but looks like I didn't: using background jobs is a great way to control the pressure given by webhooks during a flash sale, so encouraging this is great!